steam_app/controllers/GamesViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ViewController.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 28/12/22.
+//
+
+import UIKit
+
+
+let gamesList = LocalCache.gamesList
+
+class GamesViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureTableView()
+    }
+    
+    func configureTableView() {
+        let nib = UINib(nibName: GamesTableViewCell.nibName, bundle: nil)
+        
+        tableView.register(nib, forCellReuseIdentifier: GamesTableViewCell.nibName)
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+}
+
+extension GamesViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        print("tapping on cell")
+    }
+}
+
+
+extension GamesViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return gamesList.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: GamesTableViewCell.nibName, for: indexPath) as! GamesTableViewCell
+        cellConfiguration(cell: cell, indexPath: indexPath)
+        
+        return cell
+    }
+    
+    func cellConfiguration(cell: GamesTableViewCell, indexPath: IndexPath) {
+        let game = gamesList[indexPath.row]
+        
+        cell.setLabelText(text: game.title)
+        if game.isFavorite {
+            cell.setLabelTextFont(font: .boldSystemFont(ofSize: 15))
+            cell.setStarImage(string: "star.fill")
+        }
```

Comment:
A case where game is not favorite should be considered as well inside else clause. Otherwise, when table view get's scrolled, you can seem some elements with stars even though they are not favorite. Refer to [this](https://developer.apple.com/documentation/uikit/uitableview) for more info about reusability in table view

Reply:
I would add my 2 cents here, Abubakr refers to cell reuse. Whenever cell gets off screen, it gets reused and if you scroll up and down while having many cells, UI will get out of sync (because cell got reused). Please refer to [this doc](https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse) to understand better the problem.